packages/zoe/src/issuerStorage.js

```diff
@@ -175,7 +175,7 @@ export const makeIssuerStorage = () => {
     return brandToIssuerRecord.get(brand).issuer;
   };
 
-  /** @type {ExportIssuerStorage} */
+  /** @type {IssuerStorageExportIssuerStorage} */
```

Comment:
I'm happy to change it, but this is an internal function type, so we don't necessarily need it to be up to the same standards for naming as external types. Coherence is always good though. For some context, there's two ExportIssuerStorage functions, one exposed by the issuerStorage.js file, and another exposed by the `zoeStorageManager`. The issuerStorage.js function takes an array of issuers as an argument, whereas the `zoeStorageManager` has no parameters (it calls the issuerStorage.js function and supplies the issuer array). 

So this function is the IssuerStorage - ExportIssuerStorage to distinguish the type from the other function.

Reply:
I will read with this in mind. But these names really are a significant barrier to comprehension. From you explanation, I see I was already confused by these names in another way: I just assumed, without checking, that a `*exported*` was a thing visible from outside the module. Using `*export*` for something internal is confusing. It suggests a misreading that needs to be unlearned.

Reply:
Would something more like #3201 work? Do you understand why that PR causes a type error that this one does not?

Reply:
Ok, that makes sense. I'm in the middle of fixing your PR #3201 so that we switch to using your naming, but the IssuerStorage version is distinct from the zoeStorageManager version.